tests/org.codechecker.eclipse.rcp.it.tests/src/org/codechecker/eclipse/plugin/IndicatorTest.java

```diff
@@ -0,0 +1,95 @@
+package org.codechecker.eclipse.plugin;
```

Comment:
New test for demonstrating the newly added CodeChecker stub usage in the integration tests.